tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts

```diff
@@ -293,6 +295,10 @@ describe('scalar card', () => {
       // lineChart property is now set; let the template re-render with
       // `lineChart` checks correctly return the right value.
       lineChartComponent.componentInstance.changeDetectorRef.markForCheck();
+
+      // This hack effectively resizes the line chart.
+      // Resizing the line chart will result in a cardMinMax changed event being dispatched.
+      dispatchedActions.pop();
```

Comment:
I don't get why this is needed. BeforeEach dispatchedActions should be an empty array. What is here to pop?

At what timing the line chart is resized and have an additional cardMinMax event?

Reply:
Line 297 causes the change detector to recheck the line chart component. The line chart emits events when the its view box is changed. For some reason the variable used to track whether the viewbox is changed `isViewBoxChanged` defaults to true tensorboard/webapp/widgets/line_chart_v2/line_chart_component.ts line 188. That means that the first time changes are detected the line chart emits an event saying it has been resized. Outside of tests this behavior is fine because we aren't artificially triggering it to detect changes.